src/app/modules/shared/services/logging/url-utils.ts

```diff
@@ -0,0 +1,26 @@
+import { AUTH_CALLBACK_PATH } from 'src/app/modules/auth/factories/auth-factory';
+
+export function removeOauthTokenFromUrl(url: string): string {
+  if (url?.indexOf(AUTH_CALLBACK_PATH) >= 0) {
+    const urlWithoutStateParameter = updateQueryStringParameter(url, 'state', 'TOKEN_REMOVED');
+    return updateQueryStringParameter(urlWithoutStateParameter, 'code', 'TOKEN_REMOVED');
+  }
+  return url;
+}
+
+/**
+ * Replace a specific parameter value in provided url
+ * Copied from https://stackoverflow.com/questions/5999118/how-can-i-add-or-update-a-query-string-parameter
```

Comment:
Du må vise til lisensen når du kopierer fra SO.

Reply:
Takk for tips!